eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState.hs

```diff
@@ -564,8 +606,8 @@ deriving stock instance
 instance TransUTxOState NoThunks era => NoThunks (UTxOState era)
 
 instance TransUTxOState ToCBOR era => ToCBOR (UTxOState era) where
-  toCBOR (UTxOState ut dp fs us) =
-    encodeListLen 4 <> toCBOR ut <> toCBOR dp <> toCBOR fs <> toCBOR us
+  toCBOR (UTxOState ut dp fs us sd) =
+    encodeListLen 5 <> toCBOR ut <> toCBOR dp <> toCBOR fs <> toCBOR us <> toCBOR sd
```

Comment:
If I understood @dcoutts correctly, in order to be able to implement sharing upon deserialization this incremental stake map needs to come before the utxo. Not sure what are the implications of this would be. Duncan, could you please comment on this.

Reply:
Right now there are CDDL invariants that are being maintained. If we change those, we can change the order in which things are serialized. I can see why that would be a good idea. But we have to wait for the change in the CDDL to do that.